spec/controllers/users_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    it 'returns a success response' do
+      get users_path
+      expect(response).to have_http_status(:success)
+      expect(response.body).to include('This is a list of users')
+    end
+  end
+
+  describe 'GET /show' do
+    it 'returns a success response' do
+      user = User.create!(name: 'Name', posts_counter: 0)
+      get user_path(user)
+      expect(response).to have_http_status(:success)
+      expect(response.body).to include('Here is detail of a specific user')
+    end
```

Comment:
As stated above, the project requirements expect that your Spec file should include tests that ensures correct view template is rendered.

       Example : expect(response).to render_template(:show) 

Kindly update your project accordingly to match this. :pray: